lambdipy/project_build.py

```diff
@@ -240,6 +240,8 @@ def install_non_resolved_requirements(resolved_requirements, requirements, build
     if len(packages_to_install) > 0:
         print(f'Installing remaining packages via pip:{packages_to_install}')
 
+    exclude_tests_grep = '\|'.join(exclude_tests) if exclude_tests else '*'
```

Comment:
What do you think about `exclude_tests_pattern`?

Reply:
much better